resources/css/app.css

```
@@ -9,7 +9,7 @@
 @plugin "tailwind-scrollbar-hide";
 
 @source '../views';
-@source '../ressources/js';
+@source '../resources/js';
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)